manifests/helm-profiles/compatibility-version-1.23.yaml

```diff
@@ -9,4 +9,5 @@ meshConfig:
   defaultConfig:
     proxyMetadata:
       # 1.24 behaviour changes
-      ENABLE_DEFERRED_STATS_CREATION: "false"  
+      ENABLE_DEFERRED_STATS_CREATION: "false"
+      BYPASS_OVERLOAD_MANAGER_FOR_STATIC_LISTENERS: "false"
```

Comment:
For my context and understanding, what's the significance of adding this env variable here as opposed to in the pilot.env section?

Reply:
This is relevant to Pilot Agent - Not Pilot.